src/main.ts

```diff
@@ -70,6 +71,12 @@ async function run(): Promise<undefined> {
       return
     }
 
+    if (previous && onlyCreateComment) {
```

Comment:
At this point previous is always present and is unnecessary.
```suggestion
    if (onlyCreateComment) {
```